config/customization.rb.template

```diff
@@ -0,0 +1,2 @@
+ADMIN_EMAIL = "betydb@gmail.com"
```

Comment:
Would it make sense to merge this with config/additional_environment.rb instead of having 2 files that need to be modified?